pkg/reconciler/v1alpha1/clusteringress/resources/virtual_service.go

```diff
@@ -109,16 +109,19 @@ func makeVirtualServiceRoute(hosts []string, http *v1alpha1.HTTPClusterIngressPa
 			Weight: split.Percent,
 		})
 	}
-	return &v1alpha3.HTTPRoute{
+	route := &v1alpha3.HTTPRoute{
 		Match:   matches,
 		Route:   weights,
 		Timeout: http.Timeout.Duration.String(),
-		Retries: &v1alpha3.HTTPRetry{
+		AppendHeaders: http.AppendHeaders,
+	}
+	if http.Retries != nil && http.Retries.Attempts > 0 {
+		route.Retries = &v1alpha3.HTTPRetry{
 			Attempts:      http.Retries.Attempts,
 			PerTryTimeout: http.Retries.PerTryTimeout.Duration.String(),
-		},
-		AppendHeaders: http.AppendHeaders,
+		}
```

Comment:
I don't understand this change.

Reply:
This change should not have been part of the PR, removed file.